modules/common/src/main/java/org/opencastproject/security/api/JaxbRole.java

```diff
@@ -134,11 +157,15 @@ public String getDescription() {
 
   /**
    * {@inheritDoc}
-   *
-   * @see org.opencastproject.security.api.Role#getOrganization()
    */
-  public Organization getOrganization() {
-    return organization;
+  public String getOrganizationId() {
```

Comment:
Matter of taste thing: Personally, I would write

```
 if (organizationId != null) {
   return organizationId;
 } else if (organization != null) {
   return organization.getId();
} else {
  return null;
}
```

... but fine for me if you prefer the version without else if.

Reply:
I'll keep the current version.